File: LAB3/app/src/main/java/com/example/bmicalculator/MainActivity.java
package com.example.bmicalculator;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void FirstScreen(View view) {
        Intent first = new Intent(this, SecondaryActivity.class);
        startActivity(first);
    }

    public void QuizScreen(View view) {
        Intent quiz = new Intent(this, QuizActivity.class);
        startActivity(quiz);
    }

    public void GraphScreen(View view) {
        Intent quiz = new Intent(this, GraphActivity.class);
        startActivity(quiz);
    }
}

File: LAB4/app/src/main/java/com/example/bmicalculator/MainActivity.kt
package com.example.bmicalculator

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun FirstScreen(view: View?) {
        val first = Intent(this, SecondaryActivity::class.java)
        startActivity(first)
    }

    fun QuizScreen(view: View?) {
        val quiz = Intent(this, QuizActivity::class.java)
        startActivity(quiz)
    }

    fun GraphScreen(view: View?) {
        val quiz = Intent(this, GraphActivity::class.java)
        startActivity(quiz)
    }
}